src/magic.cpp

```diff
@@ -713,7 +713,7 @@ std::string spell::duration_string() const
 
 time_duration spell::duration_turns() const
 {
-    return 1_turns * duration() / 100;
+    return 1_turns * to_turns<int>( time_duration::from_moves( duration() ) );
```

Comment:
I think `time_duration::from_moves( duration() )` alone is sufficient.

Reply:
yeah i originally tried that, but sadly the operator overrides are a bit wonky.
I end up with an unambiguous *operator.

There are already two overrides, with each having one of the two params of a multiplication as a time_duration and the other as a cont T.
That in turn means that both would qualify for a multiplication of two time_durations with each other and the compiler realy isn't a fan of that.

Reply:
```suggestion
    return time_duration::from_moves( duration() );
```
This is what I mean, the multiplication is not needed.

Reply:
i think we both mean the same, but just to make 100% sure.

i think it should be return from_turns(duration()) not from_turns(duration()/100))
the reason being that the original division by 100 was followed up by a multiplication by 100 (1_turns)

Reply:
`from_turns(duration() / 100)` or `from_moves(duration()), because judging from the original code, `duration()` is in moves.

Reply:
should be resolved in the updated version